CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 2.3.0
+
+### Major changes
+* `bem-core` dependency was updated to [2.7.0](https://github.com/bem/bem-core/blob/v2.7.0/CHANGELOG.ru.md#270).
+* BH 4.x support added ([#1587](https://github.com/bem/bem-components/issues/1587)).
+* BEM-XJST 2.x support added ([#1495](https://github.com/bem/bem-components/pull/1495)).
+
+### Bug fixes
+* An issue in `checkbox` was fixed. The block didn't work in MSIE 11/Edge ([#1590](https://github.com/bem/bem-components/issues/1590)).
+* An issue in `attach` was fixed. The block didn't work in MSIE 11/Edge ([#1596](https://github.com/bem/bem-components/issues/1596)).
+* Theme-specific styles of `input`, `textarea` and `select` of the `_width_available` modifier was moved back from `common` level to `design` ([#1548](https://github.com/bem/bem-components/issues/1548)).
```

Comment:
This is kinda breaking change and there are people who use it as is was in `2.6.0`, let's move it to major section or highlight somehow.